package.json

```diff
@@ -4,8 +4,8 @@
     "postinstall": "node script/get-quasar.js",
     "clean": "rimraf output && rimraf .pulp-cache && rimraf test/tmp",
     "build": "pulp build -- --censor-lib --strict",
-    "property-tests": "pulp test --test-path test/src --main Test.Property.Main",
-    "test": "pulp test --test-path test/src --main Test.Property.Main && pulp test --test-path test/src"
+    "test-fast": "pulp test --test-path test/src --main Test.Fast",
```

Comment:
I think since one is going to be typing `npm run test-fast` a lot, anything longer than `fast` is going to be annoying :wink:. It expresses well enough that it's not comprehesive too. I did consider renaming the other tests to be under `Feature`, but regardless I thought the task names didn't need to reflect that.